### DISCUSSION TITLE
Intégrations avec Open Food Facts ?

### DISCUSSION THREAD

**Discussion by teolemon** [betagouv/ma-cantine#998] [Idées]
Hello, Pierre (Open Food Facts),
Avez-vous envisagé des intégrations montantes (faire remonter de la donnée sur les produits vers Open Food Facts grâce aux divers utilisateurs) et descendantes (récupérer de la donnée produit existante) avec Open Food Facts ?
On a: 
- nutrition (Nutri-Score…)
- ingredients
- labels (bio, responsable…)
- lieux de fabrication, origines
- eco-score
- et plus

Ainsi qu'une API Lecture/Ecriture, et des squelettes d'applis si vous voulez faire des systèmes de scan pour les cantiniers.

Si ça vous intéresse, dispo sur pierre@openfoodfacts.org ou sur https://slack.openfoodfacts.org pour en discuter

**Comment by JenniferStephan** [comment]
Hello Pierre, @teolemon 

Désolée de la réponse tardive et chouette de voir ton issue enthousiaste sur notre produit qui commence à peine :D 
Avec plaisir pour échanger sur les synergies de nos deux sujets ! 
Je reviens vers toi par mail avec des créneaux de dispo si ok pour toi 
Très bonne journée
